sdk/location/src/main/java/com/klaviyo/location/Geofencing.kt

```diff
@@ -27,7 +27,7 @@ fun Klaviyo.registerGeofencing(): Klaviyo = safeApply {
  * Klaviyo will automatically stop monitoring if location permissions are revoked.
  */
 fun Klaviyo.unregisterGeofencing(): Klaviyo = safeApply {
-    Registry.getOrNull<LocationManager>()?.startGeofenceMonitoring() ?: run {
+    Registry.getOrNull<LocationManager>()?.stopGeofenceMonitoring() ?: run {
```

Comment:
whoops!

Reply:
lol I'm glad I'm not the only one for whom negation is hard!